IntelliSOFT/Example CHT application/cht_pvers/test/forms/assessment.spec.js

```diff
@@ -14,7 +14,7 @@ describe('Assessment form test', () => {
     beforeEach(async () => {
         await harness.clear();
         // set harnes date to Jan 1st 2023
-        return await harness.setNow('2023-08-24');
+        return await harness.setNow('2024-04-03');
```

Comment:
Is there a particular reason why the date is set (and changed) to this date? Also, the comment above is misleading.

Reply:
This has been update to pick current date by default.